pkgs/by-name/re/renovate/package.nix

```
@@ -0,0 +1,93 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  nodePackages,
+  nodejs,
+  pnpm_9,
+  python3,
+  renovate,
+  testers,
+  xcbuild,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "renovate";
+  version = "37.393.0";
+
+  src = fetchFromGitHub {
+    owner = "renovatebot";
+    repo = "renovate";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-YgxcGNMgmwrausdR7kvG1NiyQPn0FcCq/isf9qUDCFY=";
+  };
+
+  postPatch = ''
+    substituteInPlace package.json \
+      --replace-fail "0.0.0-semantic-release" "${finalAttrs.version}"
+  '';
+
+  nativeBuildInputs = [
+    makeWrapper
+    nodePackages.node-gyp
+    nodejs
+    pnpm_9.configHook
+    python3
+  ] ++ lib.optional stdenv.hostPlatform.isDarwin xcbuild;
+
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-Zbe561q6xDKDIN+E/2eyQMz2GtpPvJEv2pAauMa+8pE=";
+  };
+
+  env.COREPACK_ENABLE_STRICT = 0;
+
+  buildPhase = ''
+    runHook preBuild
+
+    pnpm build
+
+    rm -r node_modules
+    pnpm install --prod --offline --ignore-scripts
```

This can be done with prune iirc
```suggestion
    pnpm prune --prod --offline --ignore-scripts
```

---

I used `pnpm install` because `pnpm prune --prod` failed, but I think I forgot to add `--ignore-scripts`.